### PR TITLE
new macro which can handle the default value

### PR DESCRIPTION
**Pull request by balditommaso**
New MACRO which allows the user to define a default value for an input argument. The idea is to speed up the testing phase of a FEMacro.

### CODE REVIEW

**Comment by balditommaso** [comment]
Now I am keeping the same macron name, but the Default is called based on on the number of arguments passed

File: otsdaq/FECore/FEVInterface.cc [hunk 1]
```diff
@@ -626,11 +626,16 @@ const std::string& FEVInterface::getFEMacroConstArgument(frontEndMacroConstArgs_
...
 //	helper function for getting the copy of the value of an argument
 template<>
-std::string ots::getFEMacroConstArgumentValue<std::string>(FEVInterface::frontEndMacroConstArgs_t& args, const std::string& argName)
+std::string ots::getFEMacroConstArgumentValue<std::string>(FEVInterface::frontEndMacroConstArgs_t& args, 
+														   const std::string& argName,
+														   const std::string& defaultValue)
 {
-	return FEVInterface::getFEMacroConstArgument(args, argName);
+	const std::string& data = FEVInterface::getFEMacroConstArgument(args, argName);
+	if (data.compare("") == 0)
```
Comment (eflumerf) [inline otsdaq/FECore/FEVInterface.cc]: Probably easier to understand as `if(data == "")`

Comment (balditommaso) [reply otsdaq/FECore/FEVInterface.cc]: Thanks for the feedback, I change it and I also change few others stuff discussed with Ryan.

**Review by rrivera747 — APPROVED** [review]
After the latest commit, this looks good to me for adding default values to FE Macros.

**Pull request merged**